.github/workflows/publish_production.yml

```diff
@@ -108,6 +108,11 @@ jobs:
           AZURE_BLOB_ACCOUNT: ${{ secrets.AZURE_BLOB_ACCOUNT_POLICE }}
           AZURE_BLOB_ACCOUNT_KEY: ${{ secrets.AZURE_BLOB_ACCOUNT_KEY_POLICE }}
         run: yarn upload
+      - name: upload uwm
+        env:
+          AZURE_BLOB_ACCOUNT: ${{ secrets.AZURE_BLOB_ACCOUNT_UWM }}
+          AZURE_BLOB_ACCOUNT_KEY: ${{ secrets.AZURE_BLOB_ACCOUNT_KEY_UWM }}
```

Comment:
Hi, 
Can you also add this part to the `publish_bundle_production.yml` file? :)